helpers/build_packages.sh

```diff
@@ -187,7 +202,11 @@ if [ "$BUILDPKG" == "1" ]; then
     if [ -z $BUILDPKG_PATH ]; then
        echo "--build requires an argument (path to package)"
     else
-        buildpkg $BUILDPKG_PATH $BUILDSPEC_FILE
+        if [[ -z "$BUILDSPEC_FILE" ]]; then
```

Comment:
I don't see the need to check if it's empty...

Reply:
Dropped